src/scenes/Partners/Detail/Tabs/Profile/PartnerDetailProfile.tsx

```diff
@@ -0,0 +1,60 @@
+import { Grid } from '@mui/material';
+import { Many } from 'lodash';
+import { makeStyles } from 'tss-react/mui';
+import { BooleanProperty } from '~/components/BooleanProperty';
+import { EditablePartnerField } from '../../../Edit';
+import { PartnerDetailsFragment } from '../../PartnerDetail.graphql';
+import { PartnerContactCard } from './PartnerContactCard';
+import { PartnerTypesCard } from './PartnerTypesCard';
+
+interface Props {
+  partner: PartnerDetailsFragment | undefined;
+  editPartner: (item: Many<EditablePartnerField>) => void;
+}
+
+const useStyles = makeStyles()(({ spacing, breakpoints }) => ({
+  cardSection: {
+    [breakpoints.down('sm')]: {
+      marginBottom: spacing(2),
+    },
+  },
+  card: {
+    // Should this be lifted to the card component? Or at least to a shared style?
+    height: '100%',
+    boxShadow: 'none',
+    backgroundColor: 'transparent',
+  },
+}));
+
+export const PartnerDetailProfile = ({ partner, editPartner }: Props) => {
+  const { classes } = useStyles();
+
+  return (
+    <>
+      <Grid container spacing={1} alignItems="center">
+        <BooleanProperty
```

Comment:
I know this is still a question on were we are putting, but currently this Grid doesn't do anything.